extensions/amp-sidebar/0.1/amp-sidebar.css

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+amp-sidebar {
+  position: fixed !important;
+  top: 0 !important;
+  max-height: 100vh !important;
+  height: 100vh;
+  max-width: 80vw !important;
+  background-color: #efefef;
+  min-width: 30vw !important;
+  overflow-x: hidden !important;
+  overflow-y: auto !important;
+  z-index: 9999 !important;
+  will-change: transform;
+  -webkit-overflow-scrolling: touch;
+}
+
+amp-sidebar[direction="left"] {
+  left: 0 !important;
+  transform: translateX(-80vw) !important;
+}
+
+amp-sidebar[direction="right"] {
+  right: 0 !important;
+  transform: translateX(80vw) !important;
+}
+
+.amp-sidebar-open amp-sidebar {
+  transform: translateX(0vw) !important;
+}
+
+.amp-sidebar-closed amp-sidebar,
```

Comment:
Sidebar is a page wide extension - unlike others - this might have to interact with other component on the page - I see it more like communicating the current state of the page to rest of the components.